### ISSUE TITLE
music titles not showing up ingame

### ISSUE THREAD

**Issue by Smash17** [Deinonychus71/Sma5hMusic#105]
As stated, the titles of custom music are not there, all the slots are empty. It works fine besides that. Happend after newest Acropolis Update

**Comment by Deinonychus71** [comment]
Is that due to the new XMSBT feature? does it happen even if that's the only mod enabled?

**Comment by Smash17** [comment]
thanks for the response. 
I dont know if it´s due to that (im relativly new to smash modding and have no idea what this XMSBT feature is). And yes, it does happen when its the only mod enabled.

**Comment by GarnetSunset** [comment]
Same issue here :(

**Comment by DShip91** [comment]
Even as we near the end of 2024, this is still an issue. This tool hasn't been updated in two and a half years and I'm not sure if anyone is even looking at these issues.

**Comment by RandomAnimeGamer** [comment]
Yep even in 2025 no update, mod authors on anything that uses this tool just write you off as having a mod conflict. Clearly nobody gives a shit anymore

**Comment by Deinonychus71** [comment]
It has nothing to do with "not giving a shit" and everything to do with priorities in life and very limited time in general.

This tool is clearly not up to date with latest community developments. Back when this was developed there was no XMSBT, ARCropolis was less advanced and more importantly (for me) I had a lot more time.

So, I'm not going to pretend this couldn't be improved. It could, and should, be improved. And it would really benefit in supporting plugins (so songs db can be loaded on the fly etc).

I looked at it back in December 2024 when I had a bit of time. My goal was to simplify it, using plugins to automate certain tasks which in turn would help get rid of a lot of user errors etc., but ultimately found out that It would take me more time than I had to make the changes needed to get where it needs to be. That, and I would need time to ramp up on Rust, which was the other problem.

I know this isn't the answer you're looking for, but I don't wish to make promises that I likely won't be able to keep. I would like to get back to it someday, but it won't be until I know I have enough time dedicated for it.

For this specific problem however, I know for a fact that it can work with the newest version of Arcropolis because I've seen it work as recently as December 2024. Part of what's so tricky with these issues (and why I wanted to use plugins) is that a lot can happen during the building process on PC.
I would advise to try a simple test case scenario. 1 song, no special characters nothing, no space, try to see if that shows up. Load up the MSBT and see if the string is there.

**Comment by ghost** [comment]
Hey just to put it out there, I support your life changing and things falling apart, and I throw no shade, I just wish it was working, but of course that's me hoping for free work from a dev who did so much work for free. Sorry RAG gave you a really callused and mean response. much love dude, thank you for the tool!

**Comment by RandomAnimeGamer** [comment]
Two people who entirely missed the point of what I said. Y'all make it seem like I made it personal to specifically this dev, I'm talking on an overall perspective. I wasn't being mean to the core dev, but now I WILL be mean to GarnetSunset and say "if you read what I actually said, I was referring to the mod creators that write you off as having a conflict without checking". Very ironically, you did not give a shit to read the post properly.

**Comment by Deinonychus71** [comment]
I understand the frustration on all sides.

I was just trying to at least provide some closure and reasons as to why there is currently no progress.

Even though I know it doesn't help solving your issue, I will reiterate that I do not think the acropolis update is responsible for the issue you're having.
If you can come up with an example of one song mod that is not behaving as it should (no labels) please zip it and send the mod folder, and I'll see if I can at least reproduce (when time allows).

**Comment by RandomAnimeGamer** [comment]
Believe me, most of my frustration comes from the mod creators not doing much, if anything, to report the issue or mitigate it with a workaround.

These 3 mods individually have the same issue
https://gamebanana.com/mods/329584
https://gamebanana.com/mods/535770
https://gamebanana.com/mods/547401

Googling intensively, I found barely anything to fix this, but I assume it relates to something UI-side, maybe localization?

**Comment by Deinonychus71** [comment]
It could absolutely be localization, as this wasn't heavily tested tbh. All the people willing to help me test used the US version of the game.

With a quick to reproduce test case I think I should be able to at least pinpoint what is going wrong.

**Comment by ghost** [comment]
> Two people who entirely missed the point of what I said. Y'all make it seem like I made it personal to specifically this dev, I'm talking on an overall perspective. I wasn't being mean to the core dev, but now I WILL be mean to GarnetSunset and say "if you read what I actually said, I was referring to the mod creators that write you off as having a conflict without checking". Very ironically, you did not give a shit to read the post properly.

Just read this, for the record I do not appreciate your condescending behavior, even if your point is valid. Please be kinder to those you interact with and you won't be misinterpreted as someone lame.

**Comment by RandomAnimeGamer** [comment]
> Just read this, for the record I do not appreciate your condescending behavior, even if your point is valid. Please be kinder to those you interact with and you won't be misinterpreted as someone lame.

The fact you interpreted my first message as condescending behavior speaks miles to your own bias towards text on a screen. This is your cue to look inward and do better.

**Comment by ghost** [comment]
> > Just read this, for the record I do not appreciate your condescending behavior, even if your point is valid. Please be kinder to those you interact with and you won't be misinterpreted as someone lame.
> 
> The fact you interpreted my first message as condescending behavior speaks miles to your own bias towards text on a screen. This is your cue to look inward and do better.

Excruciatingly petty and very unhelpful, have a nice life, I will block you now 👍

You are projecting your own insecurities onto a man who is quite satisfied in his life. Thanks for the unwarranted therapeutic advice though.  <3

**Comment by RandomAnimeGamer** [comment]
> Excruciatingly petty and very unhelpful, have a nice life, I will block you now 👍
> 
> You are projecting your own insecurities onto a man who is quite satisfied in his life. Thanks for the unwarranted therapeutic advice though. <3

Lmao all I did was tell you one finger my way points 3 in yours. I don't care if you block me, you're the one that got petty over perceived tone on text on a github issue. 

Your attitude has no place in the dev sphere. Too many people averse to actual feedback like you. Have a nice life

**Comment by N0TM4NU** [comment]
Once you made the build, go in the ArcOutput folder, then in ui > message and rename the "msg_bgm+us_en.msbt" and the "msg_title+us_en.msbt" to "msg_bgm.msbt" and "msg_title.msbt"
